packages/file/pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: file
-version: 6.1.4
+version: 6.2.0
 description:
   A pluggable, mockable file system abstraction for Dart. Supports local file
   system access, as well as in-memory file systems, record-replay file systems,
   and chroot file systems.
 repository: https://github.com/google/file.dart/tree/master/packages/file
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=3.0.0-0 <4.0.0'
```

Comment:
Fwiw this constraint will need to be more specific, something like `>=3.0.0-321.0.dev <4.0.0`. However, I am not totally sure how that would interact with the _beta_ channels, I don't know what their versions look like.

Reply:
I think 3.0.0-0 works, but won't work with, say 319 lol... Does it need to be "that" gradual?

Reply:
Flutter is using 3.0.0-0